cost-model/src/cost_tracker.rs

```diff
@@ -83,10 +85,18 @@ impl Default for CostTracker {
             transaction_signature_count: 0,
             secp256k1_instruction_signature_count: 0,
             ed25519_instruction_signature_count: 0,
+            transaction_cost_histogram: Histogram::new(),
         }
     }
 }
 
+#[cfg(RUSTC_WITH_SPECIALIZATION)]
+impl solana_frozen_abi::abi_example::AbiExample for CostTracker {
+    fn example() -> Self {
+        Self::default()
```

Comment:
this field is not serialized or deserialized; we don't need to care about a stable abi for it!